en/contributing-build.md

```diff
@@ -0,0 +1,52 @@
+Title: Build the Docs
+TODO: can add more user-friendly instructions on setting up local web server
+      talk about entr utility
+
+
+# Build the Docs
+
+Every non-trivial contribution must first have its HTML built and verified
+before a pull request (PR) is made from it.
+
+First install the builder. On Ubuntu 16.04 LTS:
+
+```bash
+sudo snap install documentation-builder
+```
+
+To build the HTML, while in the root of the MAAS docs repository:
+
+```bash
+/snap/bin/documentation-builder
```

Comment:
I know this is accurate, but do you think it may be better to trust that the snap adds documentation-builder to $PATH?